site/source/pages/examples/bar-map-integration.hbs

```diff
@@ -15,10 +16,10 @@ layout: example.hbs
 
 {{#markdown}}
 ```html
-{{>example-code-header}}
+{{>example-code-header-amd}}
```

Comment:
I think in #96 you get rid of the code example header/footer, but I didn't realize that until after I created the example AMD header. If you want I can get rid of it which might help avert merge conflicts.